src/lib.rs

```diff
@@ -459,16 +459,22 @@ impl<K: IntKey, V> IntMap<K, V> {
     }
 
     #[inline]
-    fn ensure_load_rate(&mut self) {
+    fn increase_cache_if_needed(&mut self) -> bool {
```

Comment:
This rename isn't strictly needed, but exposing the result is (for the fastest path, anyway), and I felt the new name made the polarity of the boolean a *little* clearer. There's definitely room for bikeshedding though. Or an internal doc comment.

Reply:
I like the new name.